_data/simple-icons.json

```
@@ -9370,6 +9370,11 @@
             "hex": "1A1928",
             "source": "https://www.pioneerdj.com"
         },
+        {
+            "title": "Piped",
+            "hex": "D43225",
```

```suggestion
            "hex": "F84330",
```
Color picked from one end of the gradient. We can also do the dominant color
```suggestion
            "hex": "D83427",
```